.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           go-version: 1.18
 
       - name: run docker image
-        run: docker run -d -p 9944:9944 threefolddev/tfchain-dev:2.2.0-rc5 --dev --ws-external
+        run: docker run -d -p 9944:9944 dylanverstraete/tfchain:2.2.0-rc3 --dev --ws-external
```

Comment:
Not sure if this is correct?

Reply:
No it's not. We need a new build once tfchain revert has been completed.

Reply:
I will create a docker image and give you the tag